test/test_conversion_driver_esm1p5.py

```diff
@@ -111,20 +116,43 @@ def _mock_process_with_exception(error_message):
     "input_list", [[], ["fake_file"], [
         "fake_file_1", "fake_file_2", "fake_file_3"]]
 )
-def test_convert_fields_file_list_success(mock_process, input_list):
+def test_convert_fields_file_list_success_delete(mock_process,
+                                                 mock_os_remove,
+                                                 input_list):
+    """
+    Test that process is called for each input and that each input is deleted.
+    """
     input_list_paths = [Path(p) for p in input_list]
+
     succeeded, _ = esm1p5_convert.convert_fields_file_list(
-        input_list_paths, "fake_nc_write_dir")
+        input_list_paths, "fake_nc_write_dir", delete_ff=True)
```

Comment:
`delete_ff=True` causes a test failure as `convert_fields_file_list()` has no flag arg.

Reply:
Yep, sorry about this – forgot to fix the tests after making the change. Should be fixed now.